src/context/AppContext.tsx

```diff
@@ -6,8 +6,10 @@ export type AppState = {
   name: string;
 };
 
+const userColorScheme = window.matchMedia('(prefers-color-scheme: dark)') ? 'dark' : 'light';
```

Comment:
this variable always is true.
you should check if it matches or not.
`
window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'light'
`